introduction/week-1/lesson.md

```diff
@@ -0,0 +1,69 @@
+# Introduction 1
```

Comment:
I'm don't think this should be moved across since this is London specific

Reply:
Hey @ChrisOwen101 , is it the whole `introduction` folder you want to remove? And if so, did you want to keep a reference of this folder anywhere (e.g. in the london branch) or are you happy for it to be gone forever?

Reply:
Let's keep it in the London branch - just don't bring it to master :)